moto/events/responses.py

```diff
@@ -409,3 +409,48 @@ def cancel_replay(self):
         result = self.events_backend.cancel_replay(name)
 
         return json.dumps(result), self.response_headers
+
+    def create_connection(self):
+        name = self._get_param("Name")
+        description = self._get_param("Description")
+        authorization_type = self._get_param("AuthorizationType")
+        auth_parameters = self._get_param("AuthParameters")
+
+        if authorization_type != "API_KEY":
+            raise NotImplementedError(
```

Comment:
Do we need this check? It doesn't look like we're doing anything with this data anyway, we just store it and return it later.
So which authorization-type is passed in doesn't really matter